service/src/main/java/bio/terra/pipelines/app/controller/GlobalExceptionHandler.java

```diff
@@ -128,9 +129,14 @@ public ResponseEntity<ApiErrorReport> catchallHandler(Exception ex) {
 
   private ResponseEntity<ApiErrorReport> buildApiErrorReport(
       @NotNull Throwable ex, HttpStatus statusCode, List<String> causes) {
+    // only logging 5** errors to sentry, should we log more/every error?  Other services seem to do
```

Comment:
nit: we can probably remove the question here

Reply:
yup yup this was for you to see it :). will remove